traffic_ops/app/lib/UI/ConfigFiles.pm

```diff
@@ -312,6 +312,9 @@ sub param_data {
 		if ( $row->parameter->name eq "location" ) {
```

Comment:
tbh, when I see this kind of stuff I start to think we've overloaded the parameter table. parameters with config_file != null should imo be parameters that are written to that config file but for these 2 special cases (location and scope) we key off name to skip over them.

it almost feels like a configfiles table could be justified with name, location, scope....but we can hold that thought for now...